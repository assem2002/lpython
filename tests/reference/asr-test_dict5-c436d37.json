{
    "basename": "asr-test_dict5-c436d37",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_dict5.py",
    "infile_hash": "4b1b45fc39df2a77ce477a006b9316550569af0309b0ac99562acf30",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_dict5-c436d37.stderr",
    "stderr_hash": "3931655b3967ece317bb4f759df5e31fa8970bc34b740a2ad187cce5",
    "returncode": 2
}