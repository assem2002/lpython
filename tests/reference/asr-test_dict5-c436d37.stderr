semantic error: Type mismatch in assignment, the types must be compatible
 --> tests/errors/test_dict5.py:3:5
  |
3 |     d = {1: "a", 2: "b"}
  |     ^   ^^^^^^^^^^^^^^^^ type mismatch ('dict[i32, i32]' and 'dict[i32, str]')


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
